frontend/src/app/libs/sign-in/sign-in.component.html

```diff
@@ -0,0 +1,15 @@
+<form [formGroup]="signIn">
+
+  <label>
+    Email
+    <input type="email" formControlName="email" required autofocus>
```

Comment:
please use angular material components, add necessary to common-components lib and go on :)